lib/src/header/Header.tsx

```diff
@@ -120,30 +124,40 @@ const DxcHeader = ({
         {isResponsive && responsiveContent && (
           <MainContainer>
             <ChildContainer padding={padding}>
-              <HamburguerItem tabIndex={tabIndex} underlined={underlined} onClick={handleMenu}>
+              <HamburguerItem tabIndex={tabIndex} onClick={handleMenu}>
                 {hamburgerIcon}
                 <HamburguerTitle>Menu</HamburguerTitle>
```

Comment:
I think we should remove `HamburguerTitle` since it is not necessary and may be semantically wrong (read [this](https://stackoverflow.com/questions/12982269/is-it-semantically-incorrect-to-put-a-div-or-span-inside-of-a-button)). The font styling can go in the above `HamburguerItem`.

Reply:
Also, I think `HamburguerTrigger` fits better than `HamburguerItem`, what do you think?

Reply:
Done!